snapcraft/cli/store.py

```diff
@@ -100,7 +100,8 @@ def _human_readable_acls(store: storeapi.StoreClient) -> str:
 @storecli.command()
 @click.argument("snap-name", metavar="<snap-name>")
 @click.option("--private", is_flag=True, help="Register the snap as a private one")
-def register(snap_name, private):
+@click.option("--store-id", metavar="<store>", help="Store ID to register with")
```

Comment:
I think `--store` is better

```suggestion
@click.option("--store", metavar="<store>", help="Store to register with")
```